lib/connection-parameters.js

```diff
@@ -88,6 +89,9 @@ ConnectionParameters.prototype.getLibpqConnectionString = function(cb) {
   if(this.database) {
     params.push("dbname='" + this.database + "'");
   }
+  if(this.replication) {
+    params.push("replication='" + (this.database === true ? "true" : this.replication) + "'");
```

Comment:
> "Replication" can have "database", "true" and "false" values.

This is checking `this.database === true`, though – not `this.replication === true`. Is that really intended? I’m not sure why `dbname='true'` (not `replication='true'`) would have any special meaning beyond a database named “true”.